### PR TITLE
Add better logging for batch update jobs

### PR DESCRIPTION
**Pull request by jcoyne**
@projecthydra/sufia-code-reviewers

screen shot 2016-04-25 at 11 07 49 am

screen shot 2016-04-25 at 11 08 00 am

Ref #1832 Ref #1779

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@jcoyne Just to confirm: still WIP?

**Comment by jcoyne** [comment]
Yes still WIP. TODO:
* Make a controller/view to show the status of a batch
* Decompose a BatchCreateJob to fan out the create jobs
* Roll up the status onto the BatchCreate
* Update notifications to direct user at the log.
* Logging for non-batch things.

**Comment by mjgiarlo** [comment]
This was blocked by a CC PR that is now merged. Also must be rebased.

**Comment by jcoyne** [comment]
@mjgiarlo Okay, ready for review.

File: app/jobs/batch_create_job.rb [hunk 1]
```diff
@@ -1,63 +1,33 @@
...
-    @saved = []
-    @success = true
+  def perform(user, titles, resource_types, uploaded_files, attributes, log)
+    log.performing!
 
     titles ||= {}
     resource_types ||= {}
 
-    create(user, titles, resource_types, uploaded_files, attributes)
-    send_user_message(user)
```
Comment (mjgiarlo) [inline app/jobs/batch_create_job.rb]: Not worth sending the user a notification about this any longer?

**Comment by mjgiarlo** [comment]
How do users get to the views in the screenshots above?

File: app/jobs/batch_create_job.rb (hunk 1 above)
Comment (jcoyne) [inline app/jobs/batch_create_job.rb]: It's moved to the BatchCreateOperation, and it occurs when all of the parts are done.

**Comment by jcoyne** [comment]
Presently you have to go directly to the URL.  I hope that we can integrate them somewhere better than that in the future.

File: app/jobs/batch_create_job.rb (hunk 1 above)
Comment (mjgiarlo) [inline app/jobs/batch_create_job.rb]: Ah, I see now. :+1:

**Comment by mjgiarlo** [comment]
@jcoyne :clap: 

I'll ticket the UI integration piece separately for 7.1.0.

**Comment by ggeisler** [comment]
In the first screenshot, the "performing" label is using the same  `.label-success` class as the "success" label. I wonder if using the `.label-info` class would be better for the "performing" state? That would visually distinguish the two different states ("performing" would have a blue label) and "info" seems appropriate for a process that's still in progress and might not actually be successful.

**Pull request merged**